models/image.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from "mongoose";
+
+// Helper variables
+const required = true;
+const unique = true;
+const { Schema } = mongoose;
+
+// Setting up image schema
+const imageSchema = new Schema({
+    originalname: { type: String, required },
+    destination: { type: String, required }, // "uploads/"
+    fieldname: { type: String, required }, // "selectedFile"
+    mimetype: { type: String, required }, // e.g. "image/jpeg"
+    filename: { type: String, required }, // e.g. "c8f44ff6c3427cdd16fcab90e3ae1377"
```

Comment:
I would make this filename and path "unique" just in case. It's practically impossible to get two with the same name, but still :)

Reply:
👍